roles/vgpu/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+vgpu_driver_path: ""
```

Comment:
```suggestion
```

Reply:
Removing this as it doesn't work with the validation and we haven't merged this change yet. So no need to maintain backwards compat anyway.